training/03_Infrastructure/02_Advanced/Adding_Users.adoc

```diff
@@ -94,8 +94,100 @@ lab_users:
 [...]
 ----
 
+=== Calling the right task
+We've just added the task and users, but we have not called it yet. In order to make the task to be called, we need to place it in the right place
+
+[source,bash]
+----
+- name: Setup lab-user
+  hosts: all
+  become: true
+  tasks:
+    - name: Setup lab-user
+      include_tasks: ./setup_lab_user.yml
+----
+
 == Setting up ssh keys
 Even if users have been created, no ssh keys have been defined, so we have not clue on how to log into the nodes machines, as we do not have a password.
 
 We want to be able to create ssh keys for our users and we need to be able to let our users to access them, easily (in case someone wants to re use our config).
 
+SSH keys configuration tasks are also listed on the `setup_lab_user.yml` file. Again, as we did with users, please add loops for all of the tasks (final result is shown below):
+
+[source,bash]
+----
+[agilpipp-redhat.com@bastion configs]$ cd a-base-config/
+[agilpipp-redhat.com@bastion a-base-config]$ vi setup_lab_user.yml
+[...]
+- name: create /home/{{ __ssh_dir }}/.ssh
+  file:
+    dest: /home/{{ __ssh_dir }}/.ssh
+    mode: 0700
```

Comment:
Let's show using symbolic mode, `u=rwx,g=,o=` rather than the octal mode. The octal mode is easy to mistake octal for decimal and also we should not expect that everyone has memorized the meanings of the numbers.

Reply:
this is what is used on the actual file that creates it on the config, I'm just documenting, you want me to change there too? It's not my config and developer is on PTO for the whole week, we could change it later when he is back

Reply:
changed to ugo